README.md

```diff
@@ -1 +1,98 @@
+[//]: <> (Do we need to include a section explaining the server connection whether that'd be McGill vpn or something else?)
```

Comment:
is this leftover from our discussion?

Reply:
Yea I left this as a reminder for the discussion.

Reply:
OK. But we wrapped up the discussion and said we wouldn't include it for now, no? So I'd remove it here so we don't wonder the same thing every time we see the raw text.

Reply:
I think we decided to have a discussion in PR(or draft) instead but anyways I had this there before bringing it up in the issue. I'll take care of it.